### ISSUE TITLE
Tests skipped on linux(3.6.13)(with no test failed) but executing on windows(3.6.10)

### ISSUE THREAD

**Issue by bmsan** [RKrahl/pytest-dependency#52]
On windows everything seems to be ok
```
platform win32 -- Python 3.6.10, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
plugins: cov-2.11.1, dependency-0.5.1
collected 14 items

14 passed, 9 warnings in 37.41s
```
But running the same tests on linux I get
```
platform linux -- Python 3.6.13, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
plugins: dependency-0.5.1, cov-2.11.1, depends-1.0.1
6 passed, 8 skipped, 13 warnings in 28.14s
```
I'm not sure why the skipping would happen as no tests failed(and why on winodws it seems to be fine)

**Comment by RKrahl** [comment]
I'd need  a little more information to know what happens.  Try to run the tests with `pytest -v -rs`.

**Comment by bmsan** [comment]
@RKrahl  Thanks for the quick reply!
After some more testing it seems I have found what is causing this(but not why):
`pytest-depends` which I had installed on the linux box but not on the windows box.
I didn't actually needed, I installed it accidentally while trying to install pytest-dependency.
After uninstalling `pytest-depends` all works ok.

Additional information: I ran with -v -rs and it seems the tests were ran out of order.
